sliceop/f64s/f64s.go

```diff
@@ -62,3 +62,24 @@ func Find(dst []int, src []float64, f func(v float64) bool) []int {
 
 	return dst
 }
+
+// Take creates a slice with all elements of src indiced by the provided indices.
+// Take uses dst as work buffer, storing elements at the start of the slice.
+// Take clears dst if a slice is passed, and allocates a new slice if dst is nil.
+func Take(dst, src []float64, indices []int) []float64 {
```

Comment:
just thinking out loud, but:
- do we want - for performances reasons (well, I guess) - to require the indices to be sorted?
- what should we do about duplicate indices? is it worrisome? do we care? (we should probably document that as well)

Reply:
Concerning the duplicated indices, I think it's better to require no duplicate - to keep it simple. Otherwise, `Take()` would have to behave more like the equivalent `numpy` function, in which the returned array has the dimension of the indices (where - obviously - arbitrary size, dimension and duplicates are allowed for `indices`). I'll make it clear in the doc that Take returns a sub-slice of `src`.